### ISSUE TITLE
(BUG) Generic Thermostats Do Not Show In 

### ISSUE THREAD

**Issue by tjclayton** [tonesto7/homebridge-smartthings#348]
## Verify the following before opening an trouble issue

**Go over all the following points, and put an `x` in all the boxes that apply.
If you're unsure about any of these, don't hesitate to ask. We're here to help!**

-  [x] That _OAuth_ is Enabled for the SmartApp under the IDE.
-  [x] The SmartApp and Device Handler are using the latest code available.
-  [x] That Both the SmartApps and Device Handlers have been _Published for You_ in the IDE.

---
## About Your Setup
-  How many devices are detected?: 12
-  Mobile App Version(Not required): iOS 1.6.49
-  SmartApp Version:v2.3.3
-  Device Handler Version: v2.3.4
-  Homebridge Version: v1.1.1
-  NodeJS Version: v12.18.3

## Expected Behavior
**Tell us what you think should be happening**
Generic Z-Wave thermostats should be available in the "Define Device Types" section so that they can be defined a either a full thermostat or heat only.

## Current Behavior
**What happens instead of the expected behavior?**
Currently, the thermostats show as Other Devices in the "Select Your Devices" section, but the thermostat options in "Define Device Types" are greyed out.

## Steps to Reproduce (for bugs)
**Provide a link to a live example, or an unambiguous set of steps to reproduce this bug. Include code to reproduce, if relevant**

1. Open Homebridge smartapp
2. Click on "Define Device Types"
3. Scroll down to Thermostats 
4. The options are greyed out

## Context
**How has this issue affected you? What are you trying to accomplish?
**Providing context helps us come up with a solution that is most useful in the real world**

While the thermostats do show as accessories, they do not appear to have full functionality (see the attached screenshot where the temperature isn't displayed correctly). It would be nice to be able to fully define them as having heat or not and having full control.

Sorry if this is a duplicate or I didn't open this correctly. I looked at the other thermostat issues and while some were close they did not seem exactly the same.

![image0](https://user-images.githubusercontent.com/21241858/89358225-ca8b8f80-d690-11ea-86e1-7c574fe70ad2.jpeg)
![image1](https://user-images.githubusercontent.com/21241858/89358227-cb242600-d690-11ea-94f8-c22ad094825f.png)
![image2](https://user-images.githubusercontent.com/21241858/89358228-cb242600-d690-11ea-96c9-983d6d537fc5.png)
![image3](https://user-images.githubusercontent.com/21241858/89358229-cb242600-d690-11ea-9e5f-c106242a7d12.png)
![image4](https://user-images.githubusercontent.com/21241858/89358230-cb242600-d690-11ea-8b1a-63c610d67307.png)
<img width="302" alt="Screen Shot 2020-08-04 at 8 24 02 PM" src="https://user-images.githubusercontent.com/21241858/89358231-cb242600-d690-11ea-8a26-0bd11af7f725.png">

---
Please include a copy of any relevant log output to assist in tracking down the bug

**Comment by Tom2Drake** [comment]
Do you have any log message regarding this thermostat in your Homebridge?

**Comment by tjclayton** [comment]
Hi, Thanks for the note. I am not sure where to go for the log in the Homebridge v2 SmartApp in the SmartThings iOS app. Can you please point me in the right direction? Sorry if this is an obvious question. That is where they are not showing up as thermostats. I try to show this in the images, but if others will help please let me know. Thank you.

**Comment by Tom2Drake** [comment]
you can find the console output from your homebridge instance when you use Homebridge Config UI X with your web browser.

**Comment by tjclayton** [comment]
Sorry. I guess that is my confusion. The smartapp logs back to the main homebridge instance?

Thanks.

**Comment by Tom2Drake** [comment]
let's make one step back. Can you add your thermostats as "other device" to the homebridge?

**Comment by tjclayton** [comment]
Yes. They are just not showing up in the thermostat section in the smartapp.

**Comment by tonesto7** [comment]
It's because they are not including basic thermostat capabilities used to determine they are a thermostat. 
I will see what I can find but my plate is pretty full right now and might take a bit

**Comment by tjclayton** [comment]
Great. Thanks very much.

**Comment by forevaclevah** [comment]
I am also trying to connect a thermostat (in my case a cielo device) and getting the same behavior. I am using generic device type in Samesung IDE but only shows up as an on/off switch with the Homebridge smart app. Would be awesome if you could fix it!
